olp-cpp-sdk-authentication/include/olp/authentication/SignInResult.h

```diff
@@ -105,7 +105,7 @@ class AUTHENTICATION_API SignInResult {
   /**
    * @brief Gets the access token expiry time in seconds.
    *
-   * @return Duration for which token stays valid.
+   * @return The expiry time of the access token (in seconds).
```

Comment:
There was a vivid discussion regarding this in some other PR. The bottom line was that because of the method returning `std::chrono::seconds` no explicit mention of the unit should be necessary for the documentation.
I agree with that.
Same applies for methods that take in `std::chrono::seconds`.

Reply:
I made the necessary changes.